library/src/lib/combobox-input/combobox-input.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="fd-combobox-input">
+    <fd-popover [arrow]="false">
+        <fd-popover-control>
+            <div
+                class="fd-combobox-control"
+                aria-label="Image label"
+                aria-controls="F4GcX348"
+                aria-expanded="false"
+                aria-haspopup="true"
```

Comment:
Do we need aria-controls and aria-expanded since we are using `fd-popover-control`?